scripts/sdk-api-diff-request-tempalte.yml

```diff
@@ -0,0 +1,16 @@
+key.request: source.request.editor.open.interface
+key.name: "5F63C5B8-6D92-44FF-8012-DCA7D787D243"
```

Comment:
I think this should be in env var or at least sample value and user will have to update.

Reply:
But why update that value? It's just a dummy value

Reply:
ok, maybe we can use "XXXXXX..." or something like "your-id..."

Reply:
Changed to env var